aries-site/src/data/wcag/components.json

```diff
@@ -48,5 +48,419 @@
+      {
+        "rule": "1.4.7",
+        "label": "Low or No Background Audio",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.8",
+        "label": "Visual Presentation",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "not-applicable",
+        "notes": "Not directly applicable to the card, but is applicable to its contents."
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "conditional",
+        "notes": "Dependent on the content of the card and whether the card is interactive."
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.13",
+        "label": "Content on Hover or Focus",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.1.1",
+        "label": "Keyboard",
+        "status": "conditional",
+        "notes": "Dependent on the card's content and interactivity. If the entire card is a link, use an <a> element. Test that the card and any interactive elements within it can be reached and activated using the keyboard alone."
+      },
+      {
+        "rule": "2.1.2",
+        "label": "No Keyboard Trap",
+        "status": "conditional",
+        "notes": "Requires implementor to test depending on the card's content."
+      },
+      {
+        "rule": "2.1.3",
+        "label": "Keyboard (No Exception)",
+        "status": "conditional"
+      },
+      {
+        "rule": "2.1.4",
+        "label": "Character Key Shortcuts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.1",
+        "label": "Timing Adjustable",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "not-applicable",
+        "notes": "I don't think hover elevation is considered here. Check with team."
```

Comment:
Kinda on the fence about this, agree it would be good to chat as a team. Maybe this is something we could bring up in our call Friday. Did a quick google search about if a hover elevation is considered an animation and here is what I got back.
<img width="1151" alt="Screenshot 2025-03-06 at 1 02 12 PM" src="https://github.com/user-attachments/assets/706baa62-8575-4fae-90cc-3c26ab1185a0" />

We don't do anything in grommet with 'prefers-reduced-motion', feel like this is an area across grommet that we could improve upon

Reply:
Marked as "failed" and expanded on rationale in notes.